test/parallel/test-crypto-dh.js

```diff
@@ -22,6 +22,28 @@ assert.strictEqual(secret2.toString('base64'), secret1);
 assert.strictEqual(dh1.verifyError, 0);
 assert.strictEqual(dh2.verifyError, 0);
 
+{
+  const DiffieHellman = crypto.DiffieHellman;
+  const dh = DiffieHellman(p1, 'buffer');
+  assert(dh instanceof DiffieHellman, 'DiffieHellman is expected to return a ' +
+                                      'new instance when called without `new`');
+}
+
+{
+  const DiffieHellmanGroup = crypto.DiffieHellmanGroup;
+  const dh = DiffieHellmanGroup('modp5');
+  assert(dh instanceof DiffieHellmanGroup, 'DiffieHellmanGroup is expected to' +
+                                           ' return a new instance when ' +
+                                           'called without `new`');
+}
+
+{
+  const ECDH = crypto.ECDH;
+  const dh = ECDH('prime256v1');
```

Comment:
Nit: how about changing the variable name to `ecdh`?

Reply:
Thank you for your review.
I fixed variable names.